source/lovelace/changelog.markdown

```diff
@@ -9,41 +9,49 @@ sharing: true
 footer: true
 ---
 
+## Changes in 0.73.1
+- Setting Lovlace as default updated Overview button to point to lovelace
```

Comment:
typo, Lovelace. Also we should consistently capitalize Lovelace (at end of sentence)